Example/Example/ViewController.swift

```diff
@@ -219,7 +219,23 @@ class RowsExampleViewController: FormViewController {
                         $0.value = 👦🏼
                         $0.selectorTitle = "Choose an Emoji!"
                     }
-            
+
+                <<< SectionedPushRow<String, Emoji>() {
```

Comment:
I would prefer to use Strings always as key type (K generic type) so we don't need to specify it.